chapters/extend/classifier.rst

```diff
@@ -0,0 +1,35 @@
+==================
```

Comment:
Can you add summary of the topics that will be covered? something like this

.. contents:: Table of Contents
    :depth: 2

Reply:
👍